### PR TITLE
rhub release

### PR DESCRIPTION
**Pull request by maelle**
* wait until there's a newer new version, with the `cran_summary()` fix

* when ready change date

### CODE REVIEW

**Comment by maelle** [comment]
Dependent on https://github.com/r-hub/rhub/issues/240

**Comment by maelle** [comment]
@gaborcsardi if you approve this PR we can publish it today, or tomorrow (in which case I'll change the date in the filename and in the post).

File: content/post/2019-04-08-rhub-release.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+### Local Docker checks! Not for Windows yet :crying_cat_face:
+
+Two new functions help you make the most of R-hub Docker Linux images:
+
+* `local_check_linux_images()` lists R-hub Docker images.
+* `local_check_linux()` allows to run an R-hub check on the local
+  host's Docker. The function is doesn't work on Windows yet, which we plan to solve in the next CRAN release.
+  
+Find more information in the dedicated vignette [Local Linux checks with Docker](https://r-hub.github.io/rhub/articles/local-debugging.html).
```
Comment (gaborcsardi) [inline content/post/2019-04-08-rhub-release.md]: Maybe write here explicitly that this currently works on Linux and macOS.

Comment (gaborcsardi) [reply content/post/2019-04-08-rhub-release.md]: Just to avoid thinking that it only works on Linux.

**Comment by maelle** [comment]
Ok, fixed. What's your favorite publication date, now or tomorrow? Both are fine with me.

**Comment by gaborcsardi** [comment]
I don't mind, up to you.

**Comment by gaborcsardi** [comment]
It is already on the RStudio mirror, but it will take 1-2 days to get binaries on macOS and win.

**Comment by gaborcsardi** [comment]
But it does not have compiled code, so it is not important to have binaries....

**Comment by maelle** [comment]
Ok, I'll publish it now then, because I'm planning another post for later this week, and it's good if they aren't too close together.

**Pull request merged**